concept/type_/role_type.rs

```diff
@@ -96,26 +97,26 @@ impl<'a> RoleType<'a> {
 
     pub fn get_annotations<'m>(
         &self,
-        type_manager: &'m TypeManager<'_, impl ReadableSnapshot>,
+        type_manager: &'m TypeManager<impl ReadableSnapshot>,
     ) -> Result<MaybeOwns<'m, HashSet<RoleTypeAnnotation>>, ConceptReadError> {
         type_manager.get_role_type_annotations(self.clone().into_owned())
     }
 
-    pub fn set_annotation(&self, type_manager: &TypeManager<'_, impl WritableSnapshot>, annotation: RoleTypeAnnotation) {
+    pub fn set_annotation(
```

Comment:
@flyingsilverfin , I reverted this to `pub` from `pub(crate)` because a test wasn't compiling. There aren't usages outside test. Does this need to be `pub(crate)`?